sdk/vision/azure-ai-vision-imageanalysis/README.md

```diff
@@ -0,0 +1,342 @@
+
+### Analyze from image buffer or URL
+
+The `ImageAnalysisClient` has two overloads for the method `analyze`:
+* Analyze an image from an input [bytes](https://docs.python.org/3/library/stdtypes.html#bytes-objects) object. The client will upload the image to the service as part of the REST request.
+* Analyze an image from a publicly-accessible URL. The client will send the image URL to the service. The service will fetch the image.
+
+The examples below show how to do both. The `analyze` from an input `bytes` object examples populate the `bytes` object by loading an image from a file on disk.
+
+### Supported image formats
+
+Image Analysis works on images that meet the following requirements:
+* The image must be presented in JPEG, PNG, GIF, BMP, WEBP, ICO, TIFF, or MPO format
+* The file size of the image must be less than 20 megabytes (MB)
+* The dimensions of the image must be greater than 50 x 50 pixels and less than 16,000 x 16,000 pixels
+
+
+## Examples
+
+The following sections provide code snippets covering these common Image Analysis scenarios:
+
+* [Generate an image caption for an image file](#generate-an-image-caption-for-an-image-file)
+* [Generate an image caption for an image URL](#generate-an-image-caption-for-an-image-url)
+* [Extract text (OCR) from an image file](#extract-text-from-an-image-file)
+* [Extract text (OCR) from an image URL](#extract-text-from-an-image-url)
+
+These snippets use the synchronous `client` from [Create and authenticate the client](#create-and-authenticate-the-client).
+
+See the [Samples](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/vision/azure-ai-vision-imageanalysis/samples) folder for fully working samples for all visual features, including asynchronous clients.
+
+### Generate an image caption for an image file
+
+This example demonstrates how to generate a one-sentence caption for the image file `sample.jpg` using the `ImageAnalysisClient`. The synchronous (blocking) `analyze` method call returns an `ImageAnalysisResult` object with a `caption` property of type `CaptionResult`. It contains the generated caption and its confidence score in the range [0, 1]. By default the caption may contain gender terms such as "man", "woman", or "boy", "girl". You have the option to request gender-neutral terms such as "person" or "child" by setting `gender_neutral_caption = True` when calling `analyze`.
+
+Notes:
+* Caption is only available in some Azure regions. See [Prerequisites](#prerequisites).
+* Caption is only supported in English at the moment.
+
+<!-- SNIPPET:sample_caption_image_file.caption -->
+
+```python
+# Load image to analyze into a 'bytes' object
+with open("sample.jpg", 'rb') as f:
+    image_buffer = bytes(f.read())
```

Comment:
f.read() should already return bytes

Reply:
Thanks @kristapratico , I removed "bytes" from "bytes(f.read())" in sample code, tested, and re-generated the snippets in the README.md.